src/reducers/StakingReducer.ts

```diff
@@ -0,0 +1,157 @@
+import { combineReducers, Reducer } from 'redux'
+
+import { StakePlugin, StakePolicy, StakePosition } from '../plugins/stake-plugins/types'
+import { Action } from '../types/reduxTypes'
+
+export type StakingAction =
+  | {
+      type: 'STAKING/ADD_POLICY'
+      walletId: string
+      stakePolicy: StakePolicy
+    }
+  | {
+      type: 'STAKING/FINISH_LOADING'
+      walletId: string
+    }
```

Comment:
I don't think this action type is used anymore, and same for many of the others. It looks like you really just to setup, update, and update_locked_amount now.

Reply:
This action is used for loading empty staking data (no staking enabled) it's also used to end re-loading started by START_LOADING independently from UPDATE_POSITION